app/views/layouts/application.html.erb

```diff
@@ -11,10 +11,18 @@
   </head>
 
   <body>
+    <% unless @env == "production" %>
+      <header class="page-header">
```

Comment:
This should really be inside the pre-existing `header` element (you’re only meant to have one per HTML “section” — this is, `article`, `aside`, `nav`, `section` elements). Otherwise we should make it a plain old `div`.